app/components/Machines.tsx

```diff
@@ -1,26 +1,52 @@
 import { Link as ChakraLink } from "@chakra-ui/next-js";
-import { Container, Heading, Box } from "@chakra-ui/react";
+import { Container, Heading, Box, Flex } from "@chakra-ui/react";
+import { PageQuery } from "@/tina/__generated__/types";
 
 import "slick-carousel/slick/slick-theme.css";
 import "slick-carousel/slick/slick.css";
 
 import MachineCarousel from "./MachineCarousel";
+import ContentWrapper from "./ContentWrapper";
+
+type Block = NonNullable<NonNullable<PageQuery["page"]["blocks"]>[number]>;
+
+export type MachineCardsBlock = Extract<
+  Block,
+  { __typename: "PageBlocksMachineCarousel" }
+>;
+
+export default function Machines({
+  width,
+  ...props
+}: { width: number } & MachineCardsBlock): JSX.Element {
+  const adjustedWidth = width <= 1728 ? width * 0.9 : 1728 * 0.9;
 
-export default function Machines({ width }: { width: number }) {
-  const adjustedWidth = width * 0.9;
   return (
-    <Box maxW={width} backgroundColor="white">
-      <Container maxW={adjustedWidth} bg="white" py="4rem">
-        <Heading color="black" textAlign="center" mb={4} pb={2}>
-          {"Our Machines"}
-        </Heading>
-        <MachineCarousel />
-        <Container maxW={"250px"} mt={20}>
-          <ChakraLink href="/view-machines" variant="buttonPrimaryLight">
-            View Machines
-          </ChakraLink>
+    <Flex justifyContent="center">
+      <ContentWrapper>
+        <Container maxW={adjustedWidth} bg="white" py="4rem" px="0px" mx={8}>
```

Comment:
Apparently, Container component has mx set to auto by default, so I had to overwrite that here